src/uu/dmesg/src/dmesg.rs

```diff
@@ -217,3 +435,57 @@ impl Record {
         }
     }
 }
+
+impl TryFrom<u32> for Level {
+    type Error = Box<dyn UError>;
+
+    fn try_from(value: u32) -> UResult<Self> {
+        let priority = value & 0b111;
+        match priority {
+            0 => Ok(Level::Emerg),
+            1 => Ok(Level::Alert),
+            2 => Ok(Level::Crit),
+            3 => Ok(Level::Err),
+            4 => Ok(Level::Warn),
+            5 => Ok(Level::Notice),
+            6 => Ok(Level::Info),
+            7 => Ok(Level::Debug),
+            _ => todo!(),
```

Comment:
Has this todo been overlooked or is it planned for a future PR?

Reply:
Whoops, sorry I overlooked it. Fixed it in e6364af. The original `dmesg` silently ignore unknown priority / facility and simply do not print the priority / facility name when invoked with `--decode`. I'll add the test later when adding the `--decode` option. Is that okay?

Reply:
Yes, that's fine.